auctions/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+from Bogdans_shop.models import CustomModel
+from products.models import Products
+
+AuthUserModel = get_user_model()
+
+# Create your models here.
+
+
```

Comment:
Please order a little bit those. You can keep the auto-generated comment from Django, but don't let empty spaces after it.